src/frontend/src/btc/components/convert/BtcConvertFeeTotal.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+	import { getContext } from 'svelte';
+	import { BTC_CONVERT_FEE } from '$btc/constants/btc.constants';
+	import { UTXOS_FEE_CONTEXT_KEY, type UtxosFeeContext } from '$btc/stores/utxos-fee.store';
+	import { ckBtcMinterInfoStore } from '$icp/stores/ckbtc.store';
+	import ConvertFeeTotal from '$lib/components/convert/ConvertFeeTotal.svelte';
+	import { CONVERT_CONTEXT_KEY, type ConvertContext } from '$lib/stores/convert.store';
+
+	export let totalFee: bigint | undefined = 0n;
+
+	const { sourceToken, sourceTokenExchangeRate, destinationToken } =
+		getContext<ConvertContext>(CONVERT_CONTEXT_KEY);
+
+	const { store: storeUtxosFeeData } = getContext<UtxosFeeContext>(UTXOS_FEE_CONTEXT_KEY);
+
+	$: totalFee =
```

Comment:
Can we have a test for this?

Reply:
Sure, added tests for it. Please take a look.